src/components/Text/Text.tsx

```diff
@@ -12,7 +12,7 @@ export type TextProps = {
 } & React.DetailedHTMLProps<React.HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>
 
 export const Text = React.forwardRef<HTMLHeadingElement, TextProps>(
-  ({ variant = 'body2', secondary, as, ...otherProps }, ref) => {
+  ({ variant = 't200', secondary, as, ...otherProps }, ref) => {
```

Comment:
I suggest we remove the default value here and require all usages to be explicit